pkg/cmd/release/update_releases.go

```diff
@@ -334,12 +334,13 @@ func generateRepositoriesFile(versions ...string) error {
 		versionToCfgToHash[v] = make(map[string]string)
 		for cfgKey, cfg := range cfgKeys {
 			url := fmt.Sprintf("https://binaries.cockroachdb.com/cockroach-v%s.%s.tgz", v, cfg)
+			fmt.Printf("getting %s\n", url)
 			resp, err := client.Get(context.Background(), url)
 			if err != nil {
 				return fmt.Errorf("could not download cockroach release: %w", err)
 			}
 			if resp.StatusCode != http.StatusOK {
-				return fmt.Errorf("unexpected status code %d when downloading %s", resp.StatusCode, url)
+				return fmt.Errorf("unexpected status code %d when downloading %s; consider turning off NetSkope", resp.StatusCode, url)
```

Comment:
I believe we avoid mentioning these sorts of specifics. Maybe say "disabling any web traffic monitoring tools"

Reply:
Undid this change, it looks like it was a transient issue on my machine